README.md

```diff
@@ -139,6 +139,24 @@ and pierre.casteran@labri.fr.
   and stop being maintained. We will generally [archive][archive] the
   repository rather than removing it completely though.
 
+- **What kind of permissions do the members have?**
+
+  Members of the coq-community organization have write-access to all
+  the repositories.  This permission should be used wisely: only very
+  minor fixes should be pushed without going through pull requests,
+  and pull requests should preferably be approved by the project
+  maintainer before getting merged.  Some maintainers may decide to
+  protect branches to enforce that all changes go through pull
+  requests and validate some conditions.  Maintainers are given
+  admin-access on the repositories that they maintain.  All members
+  have the permission to create or transfer new repositories, but they
+  should only do so after going through the standard process.  At all
+  times, there should be exactly three (active) owners of the
```

Comment:
Is there a specific reason for the number three here? Just interested in the rationale.

Reply:
Arbitrary number that is both as low as possible and sufficiently high to avoid any risk of getting the organization locked out because all the owners have simultaneously disappeared.